compose-builder/add-security-proxy.yml

```diff
@@ -0,0 +1,134 @@
+# /*******************************************************************************
+#  * Copyright 2024 Intel Corporation.
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  *******************************************************************************/
+
+version: '3.7'
+
+volumes:
+  edgex-init:
+  vault-config:
+  vault-file:
+  vault-logs:
+  consul-acl-token:
```

Comment:
Remove, these stay in the `add-security.yml` since used by more than proxy and this file is layered on top of `add-security.yml`